packages/components/src/ActionBar/ActionBar.component.js

```diff
@@ -141,7 +140,7 @@ function SwitchActions({ actions, left, right, center, getComponent, components
 	);
 }
 SwitchActions.propTypes = {
-	actions: PropTypes.arrayOf(PropTypes.shape(actionsShape)).isRequired,
+	actions: PropTypes.arrayOf(PropTypes.shape(actionsShape)),
```

Comment:
I removed the required because of console error/warning. This has a default props.